sql/src/main/java/io/crate/expression/operator/any/AnyLikeOperator.java

```diff
@@ -21,43 +21,122 @@
 
 package io.crate.expression.operator.any;
 
-import io.crate.metadata.FunctionImplementation;
-import io.crate.metadata.FunctionInfo;
+import io.crate.data.Input;
 import io.crate.expression.operator.LikeOperator;
+import io.crate.expression.operator.Operator;
 import io.crate.expression.operator.OperatorModule;
+import io.crate.metadata.BaseFunctionResolver;
+import io.crate.metadata.FunctionIdent;
+import io.crate.metadata.FunctionImplementation;
+import io.crate.metadata.FunctionInfo;
+import io.crate.metadata.functions.params.FuncParams;
+import io.crate.metadata.functions.params.Param;
+import io.crate.types.ArrayType;
+import io.crate.types.CollectionType;
+import io.crate.types.DataType;
+import io.crate.types.DataTypes;
+import io.crate.types.SetType;
+import io.crate.types.SingleColumnTableType;
+import org.apache.lucene.util.BytesRef;
 
+import java.util.List;
+import java.util.function.BiPredicate;
 import java.util.regex.Pattern;
 
+import static com.google.common.base.Preconditions.checkArgument;
+import static io.crate.expression.operator.any.AnyOperators.collectionValueToIterable;
 
-public class AnyLikeOperator extends AbstractAnyLikeOperator {
+public final class AnyLikeOperator extends Operator<Object> {
 
-    public static final String NAME = AnyOperator.OPERATOR_PREFIX + "like";
+    public static final String LIKE = AnyOperator.OPERATOR_PREFIX + "like";
```

Comment:
In this case we only have 2 names but maybe for "consistency" we could also use the Names like [here](https://github.com/crate/crate/pull/7160/commits/ace3e649868cbddd7e9c8a8d834edfbb35f6c9cb#diff-e8362945df85e1b896be63f01add5001R45)

Reply:
It's just a personal preference, not necessary :-)